components/gsl/Makefile

```diff
@@ -9,36 +9,39 @@
 #
 
 #
-# Copyright (c) 2013 Louis M. Picciano. All rights reserved.
+# Copyright (c) 2013-2016 Louis M. Picciano. All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include $(WS_TOP)/make-rules/shared-macros.mk
```

Comment:
No problem. Agreed, I use WS_TOP since my first PR in any subsequent include for the reason that you mentioned: only having to modify the path in one place when the component is relocated. I remember checking that for my first PR: I was not sure whether WS* variables were defined in the environment as for illumos builds or just as Makefile variables. :)